lib/bridge.js

```diff
@@ -312,6 +312,28 @@ Bridge.prototype._customiseAppservice = function() {
 
         this.addAppServicePath({
             method: "GET",
+            path: "/_matrix/app/:ver/thirdparty/location",
+            handler: function(req, res) {
+                if (req.params.ver !== "unstable" || !lookupController.parseLocation) {
+                    res.status(404).send("Cannot GET " + req.path + "\r\n");
```

Comment:
Hmyes; perhaps the version check should be separated out into a different return value. I'll pop that (into all five)